package.json

```diff
@@ -22,9 +22,17 @@
   "bugs": {
     "url": "https://github.com/amwmedia/plop/issues"
   },
+  "scripts": {
+    "test": "npm run test:instrument && ava && nyc report",
+    "test:instrument": "nyc instrument ./bin ./instrumented/bin && nyc instrument ./src ./instrumented/src && cp package.json ./instrumented"
```

Comment:
Since we are doing e2e tests on a CLI, we have to pre-instrument the files for nyc